src/windows/cheats.c

```diff
@@ -215,7 +215,7 @@ static rct_widget window_cheats_misc_widgets[] = {
 	{ WWT_DROPDOWN_BUTTON,	1,		WPL(1) - 10,			WPL(1),					YPL(4) + 8,		YPL(4) + 12,	STR_NUMERIC_DOWN,					STR_NONE },								// decrease rating
 	{ WWT_CLOSEBOX,			1,		XPL(0),					WPL(0),					YPL(5), 		HPL(5),			STR_CHEAT_WIN_SCENARIO,				STR_NONE},								// Win scenario
 	{ WWT_CLOSEBOX,			1,		XPL(1),					WPL(1),					YPL(5), 		HPL(5),			STR_CHEAT_HAVE_FUN,					STR_NONE},								// Have fun!
-	{ WWT_CHECKBOX,			1,		XPL(0),					WPL(0),					YPL(6),		HPL(6),		STR_CHEAT_NEVERENDING_MARKETING,			STR_CHEAT_NEVERENDING_MARKETING_TIP },	// neverending marketing campaigns
+	{ WWT_CHECKBOX,			1,		XPL(0),					WPL(0),					YPL(6),		HPL(6),		STR_CHEAT_NEVERENDING_MARKETING,			STR_CHEAT_NEVERENDING_MARKETING_TIP },	// never ending marketing campaigns
```

Comment:
Are we going by the dictionary? Because it says even I'm wrong and that it is "never-ending".
I try "neverending" and it gives me "never-ending".
I'm wondering what criteria we're using to base something as "correct"
I'm trying to follow dictionaries as close as possible.